types/proposal.go

```diff
@@ -34,14 +34,14 @@ type Proposal struct {
 
 // NewProposal returns a new Proposal.
 // If there is no POLRound, polRound should be -1.
-func NewProposal(height int64, round int32, polRound int32, blockID BlockID) *Proposal {
+func NewProposal(height int64, round int32, polRound int32, blockID BlockID, ts time.Time) *Proposal {
```

Comment:
@sergio-mena we should note that if we backport BFT time, then we'll need to refactor everywhere `NewProposal` is called as well

Reply:
Added comment to #2063